samples/apps/research-assistant-react/README.md

```diff
@@ -0,0 +1,88 @@
+# ARA - AutoGen Research Assistant
+
+ARA is an Autogen-powered AI research assistant that can converse with you to help you conduct research, write and execute code, run saved skills, learn new skills by demonstration, and adapt in response to your interactions.
+
+Project Structure:
+
+- _backend/_ code for the web api, served by FastAPI.
+- _frontend/_ code for the webui, built with Gatsby and Tailwind
+
+## Getting Started
+
+Install requirements:
+
+```bash
+cd backend
+pip install -r requirements.txt
+```
+
+AutoGen requires access to an LLM. Please see the [AutoGen docs](https://microsoft.github.io/autogen/docs/FAQ#set-your-api-endpoints) on how to configure access to your LLM. We recommend setting the `OAI_CONFIG_LIST` environment variable to point to your LLM config file.
+
+Also, if you plan to use bing search, set the `BING_API_KEY`
+
+```bash
+export OAI_CONFIG_LIST=/path/to/OAI_CONFIG_LIST
```

Comment:
Sample app should use config instructions that are consistent with the rest of autogen repo.

Reply:
Can you clarify what consistency issues you see?

Reply:
Sure:

### AutoGen:
Set `OAI_CONFIG_LIST` env to the content of the json with models etc
or 
Ensure there is a file called `OAI_CONFIG_LIST` in the `file_location`

### RA:
Set OAI_CONFIG_LIST env to the path of the json
Even when I set the path to `~/OAI_CONFIG_LIST` the code failed with "No api key error". That file exists on my machine.


If the current way is better, AutoGen repo should change the instructions?

Reply:
Ahhh. I think that documentation is wrong. That's what was confusing me. 

There is no condition where it should be set to the file path. ..It would never work. It only works when set to the file *contents*.